browser/ui/color/brave_color_mixer.cc

```diff
@@ -134,4 +180,19 @@ void AddBraveTorThemeColorMixer(ui::ColorProvider* provider,
 
   mixer[kColorToolbarButtonIconInactive] = {
       color_utils::AlphaBlend(kDarkToolbarIcon, kPrivateTorToolbar, 0.3f)};
+
+  mixer[kColorOmniboxBackground] = {GetLocationBarBackground(
```

Comment:
Changes in `AddBraveTorThemeColorMixer()` is redundant as `AddBravePrivateThemeColorMixer()` is called at first.